modules/nw-dns-operator.adoc

```diff
@@ -5,9 +5,9 @@
 = DNS Operator
 
 The DNS Operator implements the `dns` API from the `operator.openshift.io` API
-group. The operator deploys CoreDNS using a daemon set, creates a Service for
+group. The operator deploys CoreDNS using a daemon set, creates a service for
```

Comment:
AFAIK, we are still trying to capitalize Operator in all cases. As described in Slack by Andrea:
>that was the one exception that we didn't override w/ these new term updates, because the decision to capitalize Operator was made outside our team

With that said, just wanted to flag that _Operator_ should be capitalized throughout this PR, according to the current guidelines.